components/scream/src/share/io/scream_output_manager.cpp

```diff
@@ -213,7 +213,8 @@ void OutputManager::run(const util::TimeStamp& timestamp)
       register_dimension(filename,"time","time",0);
 
       // Register time as a variable.
-      register_variable(filename,"time","time","s",1,{"time"},  PIO_REAL,"time");
+      auto time_units="days since " + m_case_t0.get_date_string() + " " + m_case_t0.get_time_string();
+      register_variable(filename,"time","time",time_units,1,{"time"},  PIO_REAL,"time");
```

Comment:
I don't think it's correct to do this. If you do this, then you should modify how the timestamp is passed to scorpio for writing. Right now we have (at line 251 of `scream_output_manager.cpp`):
```
     pio_update_time(filename,timestamp.seconds_from(m_case_t0));
```
If you wish time units to be in days since case start, you should probably change that to
```
     pio_update_time (filename, timestamp.seconds_from(m_case_t0)/3600);
```
Or, maybe better, add a `days_from` method to TimeStamp.

Reply:
Also: are you assuming that `days from t0` will always be an integer? Or do you want to accommodate fractional values?

Reply:
days from t0 would definitely have to be a double.  if we have output every timestep it's a guarantee that we will have fractional day output.

Reply:
Yeah, I agree that days_from would be better than seconds_from. Would you mind making that change, Aaron? I couldn't find where seconds_from was defined by grepping or I would have made that change myself...

Reply:
Regarding Luca's second question of "should days from t0 be an integer?" - no. It already isn't because I was dividing seconds (which is an integer) by 86400.0 and passing the result to register_variable, which expects a PIO_REAL as an argument.

Reply:
ahhh, but I think Luca is concerned because `seconds_from` returns an integer.  I'll put in the `days_from` and it will reuturn a double.

Reply:
done

Reply:
Yeah, doing `int / int` needs care, since the result gets rounded. That's what I meant.